src/Fetch.res

```diff
@@ -82,6 +83,20 @@ module Headers = {
   external get: (t, string) => option<string> = "get"
   @send external has: (t, string) => bool = "has"
   @send external set: (t, string, string) => unit = "set"
+
+  /**
+    * Callback format is (. value, key)
+    */ @send
+  external forEach: (t, (. string, string) => unit) => unit = "forEach"
+
+  /**
+    * Callback format is (. value, key, headers)
+    */ @send
+  external forEachWithHeaders: (t, (. string, string, t) => unit) => unit = "forEach"
```

Comment:
I wonder if it wouldn't be better to combine these. Point-free style isn't very common in ReScript anyway, and it's easy enough to ignore the last argument by using `_`.

Reply:
do you have a preference for `@uncurry` vs `(.)`? I forgot to push but I have them as `@uncurry` locally.